src/Common/Clients/Clients.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { Row, Col } from "react-grid-system";
+
+import BoxedLayout from "../BoxedLayout/BoxedLayout";
+import clientsData from "../../Data/clientsData";
+
+import "./Clients.scss";
+
+export default function Testimonial() {
+  const displayClients = clientsData.map((aClient, index) => {
```

Comment:
I personally don't like naming things with "a" in front, nor have I ever actually seen it used anywhere. Wouldn't it be better to just call it "client" ?

Reply:
Ah yeah sure, I just used `aClient` because it was `clients` before. But now that it's `clientsData`, it makes sense to use `client` again. I think these prefixes can be helpful as it is very easy to confuse `clients` and `client`. This isn't something that I made up, it is a legitimate programming technique.